GVRf/Framework/backend_daydream/src/main/jni/daydream_renderer.cpp

```diff
@@ -155,15 +155,10 @@ void DaydreamRenderer::DrawFrame(JNIEnv &env) {
 
     gvr::Frame frame = swapchain_->AcquireFrame();
 
-
     gvr::ClockTimePoint target_time = gvr::GvrApi::GetTimePointNow();
     target_time.monotonic_system_time_nanos += kPredictionTimeWithoutVsyncNanos;
 
     head_view_ = gvr_api_->GetHeadSpaceFromStartSpaceRotation(target_time);
```

Comment:
I had tried it and found no difference, which is why I got rid of the calls to avoid these computations. It seems like the left/right matrices provided by daydream are just helpers that do the same thing as GVRf when it comes to calculating matrices for the left/right cameras.